python/pyarrow/ipc.py

```diff
@@ -89,10 +95,19 @@ class RecordBatchFileReader(lib._RecordBatchFileReader):
     footer_offset : int, default None
         If the file is embedded in some larger file, this is the byte offset to
         the very end of the file data
+    options : pyarrow.ipc.IpcReadOptions
+        Options for IPC serialization.
+
```

Comment:
Nit: remove this empty line?

Reply:
Removed not required empty line and updated memory_pool comment in all occurrences as requested:
38bdde3d4dd1cba26846a88f467f1af0416bf1a8